BUILDSCRIPT_MULTIPROC.bash44

```diff
@@ -216,12 +209,13 @@ OBJ=("cryptbase/o/cryptbase.o"
      "headerframe/o/statics.o"
      "stickerframe/o/statics.o")
 
+num_jobs=${#SRC[@]}
+
 ## "wait -n" was introduced in bash 4.3
 ## "${parameter@P}" was introduced in bash 4.4 (2016)
 ## both used in parallel for-loop
 # check we are using bash, and it is >= 4.4
-if [ ! -z ${BASH+x} ] && (([ "${BASH_VERSINFO[0]}" -ge 4 ] && [ "${BASH_VERSINFO[1]}" -ge 4 ]) || [ "${BASH_VERSINFO[0]}" -gt 4 ]) ; then
-  num_jobs=${#SRC[@]}
+if { [ "${BASH_VERSINFO[0]}" -ge 4 ] && [ "${BASH_VERSINFO[1]}" -ge 4 ] ;} || [ "${BASH_VERSINFO[0]}" -gt 4 ]; then
```

Comment:
This is just a nit. I couldn't figure out why `[ ! -z ${BASH+x} ]` was required, so I just removed it, and using `{ .. ;}` rather than `(..)` is a little more efficient since the former doesn't use a sub shell.